Updated info (#7866)
TheAlgorithms/Python: sorts/README.md

# Sorting Algorithms
Sorting is the process of putting data in a specific order. The way to arrange data in a specific order
is specified by the sorting algorithm. The most typical orders are lexical or numerical. The significance
of sorting lies in the fact that, if data is stored in a sorted manner, data searching can be highly optimised.

> Data representation in more readable formats is another use for sorting.

Reviewer: ```suggestion
Another use for sorting is to represent data in a more readable manner.
```
